package-lock.json

```diff
@@ -0,0 +1,1344 @@
+{
+  "name": "eslint-plugin-bestpractice",
+  "version": "1.7.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "acorn": {
+      "version": "5.7.3",
+      "resolved": "https://maven.dc1.thousandeyes.com/content/groups/npm/acorn/-/acorn-5.7.3.tgz",
```

Comment:
this shouldn't point to ThousandEyes repos since this is a public project :)

Reply:
Good point, I will change it. Also, there is no package.lock file right now. So we still want to commit the lock file right?

Reply:
We can just leave it out as part of this change